common/const.go

```diff
@@ -20,18 +20,10 @@ const (
 	// OperationalPath is the base path to store the operational details in the kvstore.
 	OperationalPath = "cilium-net/operational"
 
-	// LastFreeServiceIDKeyPath is the path where the Last free UUID is stored in the kvstore.
-	LastFreeServiceIDKeyPath = OperationalPath + "/ServicesV2/LastUUID"
```

Comment:
It's a little odd that we moved some of these keys but left others (like ServicesKeyPath). What was your reasoning behind it? (I can see the value of having constants in the package that uses them but it would be an all or nothing arrangement, and we probably put these in a common 3rd package to avoid an import loop or something).

Reply:
Hi, I only moved ones that are only used by the service, does not make sense at all to have const in a 3-party pkg.